app/controllers/SeriesDetailsController.scala

```diff
@@ -0,0 +1,32 @@
+package controllers
+
+import javax.inject.{Inject, Singleton}
+import org.pac4j.core.profile.CommonProfile
+import org.pac4j.play.scala.{Security, SecurityComponents}
+import play.api.data.Form
+import play.api.data.Forms._
+import play.api.i18n.I18nSupport
+import play.api.mvc.{Action, AnyContent, Request}
+
+@Singleton
+class SeriesDetailsController @Inject()(val controllerComponents: SecurityComponents)
+                                         extends Security[CommonProfile] with I18nSupport  {
+
+  val form = Form(
+    mapping(
+      "seriesNo" -> number
+    )(SelectedSeriesData.apply)(SelectedSeriesData.unapply)
+  )
+
+  val mockedData: Seq[(String, String)] = Seq(("series1", "Series 1"), ("series2", "Series 2"), ("series3", "Series 3"))
+
+  def seriesDetails(): Action[AnyContent] = Secure("OidcClient") { implicit request: Request[AnyContent] =>
```

Comment:
Is there a way of commonising this `Secure("OidcClient")` wrapper so we don't have to set a String every time we add  a new controller action?

Reply:
Tried to do something here but not sure if its any good

Reply:
Yep, that's the kind of thing I had in mind (assuming it's fine to use the same `Secure` instance for each request). You could also move it to its own file so it can be reused in other controllers, or we can do that later.